lib/strtoul.c

```diff
@@ -50,13 +51,13 @@ unsigned int lib_strtoul(char *nptr, char **endptr, int base)
 }
 
 
-int lib_strtol(char *nptr, char **endptr, int base)
+long lib_strtol(char *nptr, char **endptr, int base)
 {
 	int sign = 1;
 
 	if (*nptr == '-') {
 		sign = -1;
-		++nptr;
+		nptr++;
```

Comment:
Why this change? We preffer `++nptr`

Reply:
I think that in most of our codebase we use postincrementation (just my observation) so I tried to keep the code more consistent. Should we prefer one over the other when it doesn't change the logic?

Reply:
Reverted back to preinrcementation.